libs/Microsoft.MixedReality.WebRTC.Native/src/interop/global_factory.h

```diff
@@ -73,6 +130,17 @@ class GlobalFactory {
   /// Collection of all objects alive.
   std::unordered_map<TrackedObject*, ObjectType> alive_objects_
       RTC_GUARDED_BY(mutex_);
+
+  /// Shutdown options.
+  mrsShutdownOptions shutdown_options_;

```

Comment:
This seems to be uninitialized?

Also, mrsShutdownOptions is defined in interop_api.h - does it mean there's going to be an interface to set it?

Reply:
Yes, it was not in this change, but I still needed it to be defined.